tests/ui/test_login.py

```diff
@@ -1,23 +1,24 @@
+import os
 import pytest
 
 from pages.desktop.home import Home
 
 
-@pytest.mark.skip(
-    reason='https://github.com/mozilla/addons-server/issues/2462')
-def test_login(base_url, selenium, user):
+@pytest.mark.skipif(os.environ.get('PYTEST_BASE_URL') is None,
+                    reason='Live Server login currently not functioning')
```

Comment:
> Correct me if I'm blind, knowing that this is hard to fix, are there any tests currently running that are using the live-server and thus testing the overwritten live_server fixture? I can't see any but please correct me if I'm wrong.

```my_base_url``` starts the live server for each test.

> Also, this should probably use another environment variable than PYTEST_BASE_URL to skip and switch between live_server and real-environments (e.g UITEST_USE_LIVESERVER or so) to be more explicit about any implications it has on the tests.

I have a patch for this, but I will keep this in mind.